Src/libCZI/SingleChannelScalingTileAccessor.cpp

```diff
@@ -75,28 +78,27 @@ void CSingleChannelScalingTileAccessor::ScaleBlt(libCZI::IBitmapData* bmDest, fl
         const auto source = sb->CreateBitmap();
         ScopedBitmapLockerSP srcLck{ source };
         ScopedBitmapLockerP dstLck{ bmDest };
-        CBitmapOperations::CopyOffsetedInfo info;
-        info.xOffset = sbInfo.logicalRect.x;
-        info.yOffset = sbInfo.logicalRect.y;
```

Comment:
I'd suggest to leave the "old version in place, just change those two lines to 
```
        info.xOffset = sbInfo.logicalRect.x - intersect.x;
        info.yOffset = sbInfo.logicalRect.y - intersect.y;
```
c.f. [here ](https://github.com/ptahmose/libczi-zeiss/blob/a5be2c0df9e2f698f8032fe0cd2cb4f88a2c3309/Src/libCZI/SingleChannelScalingTileAccessor.cpp#L82C1-L82C1)

Reply:
Thanks for the suggestion and implementation! Looks great! :)

Reply:
I tested the changes againg with pylibczirw. The probelm is fixed with this state of the PR. From my point of viwe we are ready to merge.

Reply:
I'd also give it a go - thanks!